### PR TITLE
AWESOME BOOK PROJECT-ES6 MODULE

### PR DESCRIPTION
**Pull request by 3nityGitHub**
## In This PR:
   - I created a new repo and initialize it with git.
   - I used npm init -y command to create package.json file.
   - I  added the index.html, index.css and index.js file. 
   - I created a module folder and refactored my script using arrow function and broke them down into modules
   - I downloaded the luxon library to enable me display the current date below the navigation bar by importing it as a module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,126 @@
...
+
+- **[Responsiveness]** Using flexbox/grid to learn about responsive designs
+- **[Multiple page website:]** You are able to toggle between homepage and the aboutpage
+- **[Attractive pages]** Use of colors with mild contrast.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Source code:](https://github.com/3nityGitHub/AWESOME-BOOK-ES6-MODULE/)
+
```
Comment (wayungi) [inline README.md]: - [x] The live demo section should link to a deployed version of your project. Please include a link to your deployed project or delete this section if deployment is not a project requirement 🤝

File: Modules/bookSet.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          title,
+          author,
+        };
+
+        books.push(book);
+        localStorage.setItem('bookData', JSON.stringify(books));
+        this.displaylist();
+      }
+    }
+  }
```
Comment (wayungi) [inline Modules/bookSet.js]: - [ ] Classes should not be defined inside functions. Instead, its functions/methods are defined inside classes. Please refactor this. 😄

File: Modules/bookSet.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    addBook(title, author) {
+      if (title !== '' && author !== '') {
+        const book = {
+          id: books.length + 1,
+          title,
+          author,
+        };
+
```
Comment (wayungi) [inline Modules/bookSet.js]: - [ ] You are required to refactor all your functions to **arrow functions**. Kindly change all your functions to es6 arrow function syntax 😄

File: Modules/bookSet.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        books.push(book);
+        localStorage.setItem('bookData', JSON.stringify(books));
+        this.displaylist();
+      }
+    }
+  }
+
+  const newBook = new Book();
+  newBook.displaylist();
```
Comment (wayungi) [inline Modules/bookSet.js]: - [ OPTIONAL] Please do not instantiate the **Book** class in the module where it is defined. modules are meant to be imported and instantiated in the file where they are imported 👍🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @3nityGitHub , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @3nityGitHub 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Highlights
- App is working well :heavy_check_mark: 
- Implemented all suggested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**